src/models/SiteBanner.php

```diff
@@ -191,8 +191,8 @@ public function isActive()
             return true;
         }
 
-        $startDate = new DateTime($this->StartDate);
-        $endDate   = new DateTime($this->EndDate);
+        $startDate = new DateTime($this->StartDate ?? 'now');
```

Comment:
`new DateTime('now')` could be different to 
`new DateTime(DBDatetime::now()->Format(DBDatetime::ISO_DATETIME))`?

Reply:
Updated! Thanks